src/main/java/org/reaktivity/nukleus/tls/internal/TlsNukleus.java

```diff
@@ -207,13 +311,30 @@ static SSLContext initContext(
             TrustManager[] trustManagers = null;
             if (trustStoreFile.exists())
             {
+                trustStoreExists = true;
                 KeyStore trustStore = KeyStore.getInstance(trustStoreType);
                 trustStore.load(new FileInputStream(trustStoreFile), trustStorePassword.toCharArray());
                 // TODO: TLS Alert Record, code 112 / scope trustStore to match routes?
                 TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(
                         TrustManagerFactory.getDefaultAlgorithm());
                 trustManagerFactory.init(trustStore);
                 trustManagers = trustManagerFactory.getTrustManagers();
+
+                if (++storeIndex > 255)
+                {
+                    // cannot fit in 1 byte
+                    return null;
+                }
```

Comment:
We can probably live with this restriction for now, given that we don't need to deploy more than `256` distinct stores, but do you agree that determining the `storeIndex` could be deferred until `RESOLVE` since we only need the `storeIndex` to fit in a single byte as part of the `authorization` value now returned from `RESOLVE` ?